src/NuGet.Core/NuGet.Build.Tasks/BuildTasksUtility.cs

```diff
@@ -289,7 +289,7 @@ public static async Task<bool> RestoreAsync(
                 {
                     RestoreSummary.Log(log, restoreSummaries);
                 }
-                return restoreSummaries.All(x => x.Success);
+                return restoreSummaries;
```

Comment:
Is this fixing other bugs with restore around logging what specifically failed during a failed restore? Before, if any summary indicated failure, then the entire collection wasn't returned, therefore client couldn't tell the customer any details of what failed.

Reply:
No, it's still expected that any project that does not have a successful restore to have logged its errors.